dev/com.ibm.ws.jaxb_fat/publish/servers/jaxb_fat/schemaEE10.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:ns0="http://jaxb.web.dataobjects/" xmlns:ref="http://ws-i.org/profiles/basic/1.1/xsd" xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://jaxb.web.dataobjects/" elementFormDefault="qualified">
```

Comment:
Should these be https instead of http?

Reply:
You changed one of them to https, but not the others.  Any reason you didn't change them all to https?